app/code/Magento/AdvancedPricingImportExport/Model/Export/AdvancedPricing.php

```diff
@@ -568,10 +567,10 @@ protected function getTierPrices(array $listSku, $table)
                 if (isset($price[0]) && !empty($price[0]) || isset($price[1]) && !empty($price[1])) {
                     $select->orWhere('ap.percentage_value IS NOT NULL');
                 }
-                if (isset($updatedAtFrom) && !empty($updatedAtFrom)) {
```

Comment:
This seems out of scope from the PR.

Reply:
@antoninobonumore I agree. The way that Adobe have set up GitHub means that any files being modified must 100% pass the current coding standard(s). This change was required to appease phpstan. See the failing test here: https://github.com/magento/magento2/runs/8897630924

Reply:
This is an example of why I didn't want to remove all the `@author` tags in one go.

Reply:
Thanks for clarifying this @fredden 
This context makes this change acceptable then.

Reply:
TBH I would revert this change and add ignore in order not to break it.

Reply:
The logic isn't broken by this change. The complaint from phpstan is valid. These variables either are undefined (because they're declared inside an `if` statement further up the file) or are a string from `$this->_localeDate->date(...)->format('Y-m-d H:i:s');`, so the `isset()` check makes sense, but the `empty()` is unnecessary. The `empty()` check for `$price[...]` makes sense though, as that may be an "empty" value (like `null`).

@ihor-sviziev would you prefer that I open a separate pull request to fix this complaint from phpstan?

Reply:
Make sense! Thx for explanation